lib/app/modules/quality_reporting/measure_availability_sequence.rb

```diff
@@ -11,7 +11,7 @@ class MeasureAvailability < SequenceBase
       title 'Measure Availability'
 
       test_id_prefix 'measure_availability'
-      requires :measure_to_test
+      requires :measure_to_test, :api_key, :auth_header
```

Comment:
Will this still allow you to use Measure Availability if you do not have an `api_key` or `auth_header` i.e. if they are not needed

Reply:
yes